src/constants.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+export const notice = sprintf(
+	__(
+		'Spend only {amount} more to get free US shipping!',
+		'free-shipping-progress-bar'
+	)
+);
```

Comment:
Why is this a sprintf if you're not replacing anything?

Reply:
Also, notice should be the same one used in block.json as well, so you can move the attributes section from block.json to `registerBlockType` section.

Reply:
> Why is this a sprintf if you're not replacing anything?

I removed the obsolete `sprintf()` wrapper.

> Also, notice should be the same one used in block.json as well, so you can move the attributes section from block.json to registerBlockType section.

When moving the attributes section from `block.json` to `registerBlockType()`, I run into the following problems:
- I wasn't able to merge attributes from `block.json` and `registerBlockType()`.
- When moving all attributes from `block.json` to `registerBlockType()`, some attributes could not be detected in the frontend.

Therefore, I moved the attributes back to `block.json`.  Would you mind taking a look at this problem?